src/Normalizer/FileItemNormalizer.php

```diff
@@ -41,9 +43,29 @@ public function normalize($field_item, $format = NULL, array $context = array())
     $entity = $field_item->getEntity();
     $field_uri = $this->linkManager->getRelationUri($entity->getEntityTypeId(), $entity->bundle(), $field_name);
 
-    // Set file field-specific values.
-    $data['_embedded'][$field_uri][0]['display'] = $field_item->get('display')->getValue();
-    $data['_embedded'][$field_uri][0]['description'] = $field_item->get('description')->getValue();
+    // Add any field-specific data.
+    foreach ($this->getAdditionalProperties($field_item->getPluginId()) as $field_name) {
```

Comment:
$field_name is not a field name, it's a property name. 

Yes, target_id is special, but I'd just explicitly exclude that. so just $data['_embedded'][$field_uri][0] = $field_item->getValue() and then unset target_id.